staging/src/k8s.io/code-generator/_examples/crd/apis/test1/v1/doc.go

```diff
@@ -0,0 +1,21 @@
+/*
```

Comment:
use a better api group name please. shortnames, oneletter or something like that.